packages/core/src/lib/orchestrator/orchestrator.ts

```diff
@@ -165,6 +166,33 @@ export class Orchestrator {
         );
         verificationData = bytesToHex(signature);
         this.executeProgressHook(PROGRESS_HOOK.SEND_TX_04_02, verificationData);
+        /**
```

Comment:
This approach is wrong.

`feeLockingRequired` flag itself should be set as true in case UEA deployment statement is false.
And locking amount should be equal to requiredFunds or `0.001` PC in case that amount is negative